real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java

```diff
@@ -38,23 +38,23 @@
 	public static boolean AUTONOMUS_DEFAULT = false;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1268275770/issues/source?bid=2938665&fileBranchId=2939970#l38)